README.md

```diff
@@ -29,6 +29,19 @@ configuration at `/etc/nixos/configuration.nix`.
 }
 ```
 
+Alternatively you can define this repo directy within your `configuration.nix`:
+
+```nix
```

Comment:
Please surround with braces so this is a valid expression. Otherwise the github syntax highlighter marks things as errors.